web/src/app/selection/MaterialSelect.js

```diff
@@ -81,6 +82,10 @@ export default class MaterialSelect extends Component {
       value: value ? (multiple ? value.join(',') : value.value) : '',
     }
     const { isCleared } = this.state
+    const getValue = () => {
+      if (multiple) return
```

Comment:
Doesn't this need to return the current value?
```suggestion
      if (multiple) return value
```

Reply:
Returning the value won't clear the field.

Reply:
As-is this will result in `undefined` being passed as the `value` prop to the `Select` component in **all** cases (for multi select), clear or not. Previously we were passing the value directly, which will be an empty array if cleared.

Locally it seems to work either way without warnings, but if there is a reason `value` must always be `undefined` for multi-select, let's add a comment for future reference.

Reply:
Moving the condition to onChange works and won't return undefined.